sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectView.groovy

```diff
@@ -119,60 +109,25 @@ class ProjectView extends ProjectDesign {
     }
 
     private static void setHeaderRowStyle(HeaderRow headerRow) {
-        headerRow.getCell("Subscription").setStyleName("project-cell-min-width header-with-tooltip")
-        headerRow.getCell("SamplesReceived").setStyleName("project-cell-min-width header-with-tooltip")
-        headerRow.getCell("SamplesPassedQc").setStyleName("project-cell-min-width header-with-tooltip")
-        headerRow.getCell("LibraryPrepFinished").setStyleName("project-cell-min-width header-with-tooltip")
-        headerRow.getCell("SampleDataAvailable").setStyleName("project-cell-min-width header-with-tooltip")
-        headerRow.getCell("ProjectTitle").setStyleName("project-cell-min-width project-title-cell")
-        headerRow.getCell("ProjectCode").setStyleName("project-cell-min-width")
+        headerRow.getCell("Subscription").setStyleName("cell-min-width header-with-tooltip")
+        headerRow.getCell("SamplesReceived").setStyleName("cell-min-width header-with-tooltip")
+        headerRow.getCell("SamplesPassedQc").setStyleName("cell-min-width header-with-tooltip")
+        headerRow.getCell("LibraryPrepFinished").setStyleName("cell-min-width header-with-tooltip")
+        headerRow.getCell("SampleDataAvailable").setStyleName("cell-min-width header-with-tooltip")
+        headerRow.getCell("ProjectTitle").setStyleName("cell-min-width cell-max-width")
+        headerRow.getCell("ProjectCode").setStyleName("cell-min-width")
         headerRow.getCell("Subscription").setStyleName("subscription-cell")
     }
 
-    private void setColumnsStyle(Grid projectGrid) {
-        setComponentAlignment(projectGrid, Alignment.TOP_LEFT)
-        projectGrid.setWidthFull()
+    private static void setColumnsStyle(Grid projectGrid) {
```

Comment:
no need for static, can be non-static and no need to pass the grid, since your object knows what its components are